examples/gh-pages/scripts/components/events/ClosureListeners.js

```diff
@@ -49,6 +49,10 @@ class ClosureListeners extends React.Component {
     this.setState(this.state);
   }
 
+  _handle_infowindow_click () {
```

Comment:
Is something missing in this method? Or could it be removed aswell as the `console.log`?